array-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,11 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+let GryfArray = hogwarts.filter((student) => {
```

Comment:
Should you be using `let` to define `GryfArray`? Is it being reassigned anywhere?

Should you be using Pascal Case when defining `GryfArray`, or Camel Case?

Do you need brackets when there is only one parameter on this `filter` method?